home/src/main/resources/rdf/dynapiAbox/everytime/user_groups.n3

```diff
@@ -0,0 +1,31 @@
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl:   <http://www.w3.org/2002/07/owl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix vitro: <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#> .
+@prefix dynapi: <https://vivoweb.org/ontology/vitro-dynamic-api#> .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/user-group/editor>
+        a                       dynapi:UserGroup ;
+        rdfs:label              "Editor user group"@en-US ;
+		dynapi:name				"http://vitro.mannlib.cornell.edu/ns/vitro/authorization#EDITOR" ;
+        vitro:mostSpecificType  dynapi:UserGroup .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/user-group/self-editor>
+        a                       dynapi:UserGroup ;
+        rdfs:label              "Self editor user group"@en-US ;
+		dynapi:name				"http://vitro.mannlib.cornell.edu/ns/vitro/authorization#SELF_EDITOR" ;
+        vitro:mostSpecificType  dynapi:UserGroup .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/user-group/admin>
+        a                       dynapi:UserGroup ;
+        rdfs:label              "Admin user group"@en-US ;
+		dynapi:name				"http://vitro.mannlib.cornell.edu/ns/vitro/authorization#ADMIN" ;
+        vitro:mostSpecificType  dynapi:UserGroup .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/user-group/curator>
```

Comment:
do we need a group representing anonymous (not logged in) users?

Reply:
I didn't need it yet.